css/bourbon/README.md

```diff
@@ -0,0 +1,47 @@
+# Bourbon Docker image
+
+This image contains the sass binary and provides Bourbon.
+
+## Usage
+
+The image looks for sass files in a `/src` directory and outputs css files in a
+`/build` directory. To build your Bourbon-based sass project:
+
+    docker run \
```

Comment:
found out recently that we should be using `--rm` on most `docker run` commands to remove the intermediate container made when executing this command